[2025-04-29T17:28:37.903+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-29T17:28:37.920+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tesla_stock_sentiment_final_etl.load_sentiment_data_to_snowflake manual__2025-04-29T17:28:33.479059+00:00 [queued]>
[2025-04-29T17:28:37.929+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tesla_stock_sentiment_final_etl.load_sentiment_data_to_snowflake manual__2025-04-29T17:28:33.479059+00:00 [queued]>
[2025-04-29T17:28:37.930+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-29T17:28:37.949+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): load_sentiment_data_to_snowflake> on 2025-04-29 17:28:33.479059+00:00
[2025-04-29T17:28:37.959+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=277) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-29T17:28:37.959+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'tesla_stock_sentiment_final_etl', 'load_sentiment_data_to_snowflake', 'manual__2025-04-29T17:28:33.479059+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/tesla_stock_etl.py', '--cfg-path', '/tmp/tmpckfzj2c0']
[2025-04-29T17:28:37.961+0000] {standard_task_runner.py:72} INFO - Started process 283 to run task
[2025-04-29T17:28:37.961+0000] {standard_task_runner.py:105} INFO - Job 70: Subtask load_sentiment_data_to_snowflake
[2025-04-29T17:28:38.013+0000] {task_command.py:467} INFO - Running <TaskInstance: tesla_stock_sentiment_final_etl.load_sentiment_data_to_snowflake manual__2025-04-29T17:28:33.479059+00:00 [running]> on host 1f133dc6a737
[2025-04-29T17:28:38.091+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tesla_stock_sentiment_final_etl' AIRFLOW_CTX_TASK_ID='load_sentiment_data_to_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2025-04-29T17:28:33.479059+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-29T17:28:33.479059+00:00'
[2025-04-29T17:28:38.093+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-29T17:28:38.108+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-04-29T17:28:38.110+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2025-04-29T17:28:38.111+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-29T17:28:38.112+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-29T17:28:39.891+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-29T17:28:40.211+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-29T17:39:47.488+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-29T17:39:47.576+0000] {logging_mixin.py:190} INFO - âœ… Sentiment data loaded into dev.raw.tesla_tweet_data.
[2025-04-29T17:39:47.607+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-29T17:39:47.807+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-29T17:39:47.811+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=tesla_stock_sentiment_final_etl, task_id=load_sentiment_data_to_snowflake, run_id=manual__2025-04-29T17:28:33.479059+00:00, execution_date=20250429T172833, start_date=20250429T172837, end_date=20250429T173947
[2025-04-29T17:39:47.961+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-29T17:39:48.024+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-04-29T17:39:48.074+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-29T17:39:48.077+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
